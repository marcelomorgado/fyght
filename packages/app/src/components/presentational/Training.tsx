import React from "react";
import { Card, Button } from "antd";
import masterImage from "../../assets/img/master.png";

const { Meta } = Card;

type Props = {
  trainingCost: number;
};

export const Training = ({ trainingCost }: Props) => {
  return (
    <Card
      hoverable
      type="inner"
      cover={<img alt="example" src={masterImage} />}
      title="Train w/ Master"
    >
      <Meta
        style={{ textAlign: "center" }}
        title="Improve your Qi!"
        description=""
      />
      <p></p>
      <Button block={true} type="primary">
        +1 Qi &#926; {`${trainingCost}`}
      </Button>
    </Card>
  );
};

export default Training;
