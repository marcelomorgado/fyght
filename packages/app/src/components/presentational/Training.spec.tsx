import * as React from "react";
import { shallow } from "enzyme";

import { Training } from "./Training";

describe("Training", () => {
  test("should render the component", () => {
    expect(shallow(<Training trainingCost={1} />)).toMatchSnapshot();
  });
});
