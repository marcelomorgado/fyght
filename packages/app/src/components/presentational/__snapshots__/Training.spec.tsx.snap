// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Training should render the component 1`] = `
<Card
  cover={
    <img
      alt="master"
      src="test-file-stub"
    />
  }
  title="Train w/ Master"
  type="inner"
>
  <Meta
    description=""
    style={
      Object {
        "textAlign": "center",
      }
    }
    title="Improve your Qi!"
  />
  <p />
  <Button
    block={true}
    ghost={false}
    htmlType="button"
    loading={false}
    type="primary"
  >
    +1 Qi Îž 
    1
  </Button>
</Card>
`;
